targets/frontend/src/components/pagination/index.js

```diff
@@ -98,22 +101,19 @@ PaginationList.propTypes = {
   children: PropTypes.node.isRequired,
 };
 
-function PageButton({ currentPage, pageIndex }) {
-  const router = useRouter();
-  const qs = Object.entries(router.query)
-    .flatMap(([key, value]) => (key === "page" ? [] : `${key}=${value}`))
-    .join("&");
-
+function PageButton({ currentPage, pageIndex, onPageChange }) {
   return (
-    <Link href={`${router.route}?${qs}&page=${pageIndex}`} passHref>
-      <NavButton variant={pageIndex === currentPage ? "accent" : "secondary"}>
-        {pageIndex + 1}
-      </NavButton>
-    </Link>
+    <Button
```

Comment:
if it's a navigation, it's a link not a button

Reply:
Haha I bet you would disagree 🙃 We don't change the page in the sense that it is the same url. The only reason we put pagination in the url is to keep the state of the page in case someone copy/pastes the url. But, correct me if I'm wrong, it's all about internal state: the search, the params (the number of items, the filters, the pagination and so). Puting here a link and then ask for every page that uses this component to sync with the url to use it is, at least, weird in my opnion. It makes it very cumbersome (the page is actually very very buggy If you look at it before this PR). You sentence makes sense in general but not here I think

Reply:
since the all the search params are reflected in the url, its not an internal state anymore

Reply:
![hum](https://user-images.githubusercontent.com/705453/101336019-3ae23180-387a-11eb-96dc-50c6b253a6d5.gif)
I'm sorry but the pagination is currently not working. There do is an internal state, and currently we absolutely do not sync with the query (where is the useEffect which watches the query ?). But if you do, you will realize that have two sources of truth (the url and the state) makes it very complicated to grasp what should be done (please don't be in denial about this one). Take a quick look at the code in `informations/index.js` and look how simple it is, please do.